python/datashaper/datashaper/constants.py

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project.
+#
+"""Datashaper constants."""
+
+DEFAULT_INPUT_NAME = "source"
```

Comment:
Why not "input" and "output"?

Reply:
We could, it would require refactoring all of our fixtures though